### DISCUSSION TITLE
Problem with a jar file stored with Git LFS extension

### DISCUSSION THREAD

**Discussion by dgoncalvesramos** [community/community#21900] [New to GitHub]
<p>Hi,</p>
<p>I have an issue with a jar file that I stored with Git LFS on this <a href="https://github.com/dgoncalvesramos/qpuc-game" rel="nofollow noopener">repo</a>.</p>
<p>When I git clone the repo with git clone line command the downloaded jar file is corrupted. </p>
<p>When I download it from the Github GUI <a href="https://github.com/dgoncalvesramos/qpuc-game/blob/master/bin/qupc.jar" rel="nofollow noopener">here</a> it works perfectly.</p>
<p>The dowloaded files seems to be different since sha fingerprints doesn’t match 3ea6457e99d217caaf152e240c3c6795aeb32eea from the corrupted file I get when I git clone the repo and e471c7bc25368311899993224ce3635f4eb6307e from the file I get from the GUI. </p>
<p>Could you explain me if this is a known issue with jar files stored with Git LFS or if I did something wrong when I first push it to my GitHub repo ?</p>
<p>Regards,</p>
<p>David</p>

**Comment by dgoncalvesramos** [comment, marked as answer]
<p>I solved my issue, just had to reinstall git lfs on my new machine and run git lfs pull on my local repo just as stated <a href="https://github.com/git-lfs/git-lfs/issues/2724" rel="nofollow noopener">here</a>. </p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/dgoncalvesramos">@dgoncalvesramos</a>,</p>
<p>Just FYI, this post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. </p>
<p>Thank you for updating the Topic with the solution once you discovered it!</p>
<p>Cheers!</p>
